Classifiers/classifier-QRadar.json

```diff
@@ -0,0 +1,74 @@
+{
+	"id": "af6645d1-8209-4c7f-8458-f8edfc77b9e9",
+	"version": -1,
+	"modified": "2018-04-29T07:46:44.153394526Z",
+	"defaultIncidentType": "",
+	"mapping": {
+		"Access": {
+			"internalMapping": {
+				"Dest NT Domain": {
+					"simple": "",
+					"complex": {
+						"root": "destination_networks",
+						"filters": [],
+						"accessor": "",
+						"transformers": [
+							{
+								"operator": "general.join",
+								"args": {
+									"separator": {
+										"value": null,
+										"isContext": false
+									}
+								}
+							}
+						]
+					}
+				},
+				"Src NT Domain": {
+					"simple": "source_network",
+					"complex": null
+				},
+				"Src User": {
+					"simple": "offense_source",
+					"complex": null
+				},
+				"details": {
+					"simple": "description",
+					"complex": null
+				},
+				"occurred": {
+					"simple": "",
+					"complex": {
+						"root": "start_time",
+						"filters": [],
+						"accessor": "",
+						"transformers": [
+							{
+								"operator": "number.TimeStampToDate",
+								"args": {}
+							}
+						]
+					}
+				}
+			},
+			"dontMapEventToLabels": false
+		}
+	},
+	"custom": true,
+	"transformer": {
+		"simple": "",
+		"complex": {
+			"root": "categories",
+			"filters": [],
+			"accessor": "[0]",
+			"transformers": []
+		}
+	},
+	"keyTypeMap": {
+		"SIM User Authentication": "Access",
+		"User Login Failure": "Access"
+	},
+	"brandName": "QRadar",
+	"releaseNotes": "-"
```

Comment:
lets try to add release notes that explain a bit about the mapping/classificaiton
`QRadar classification and mapping for 'Access' incident type`